site2/docs/io-elasticsearch-sink.md

```diff
@@ -52,43 +52,44 @@ The configuration of the Elasticsearch sink connector has the following properti
 
 ### Property
 
-| Name | Type|Required | Default | Description 
```

Comment:
did you reformat the table ?

Reply:
yes it was malformed, the IDE did it. I just rendered the table locally and it looks good